README.md

```diff
@@ -19,6 +19,25 @@ This repo contains the code for generating two Docker images for Swift:
 7. Updated Dockerfiles per guidelines in [Best practices for writing Dockerfiles](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/).
 8. Added Support for amd64 and s390x architectures of Xenial Linux Ubuntu 16.04.
 
+
+
+# Quick Start
```

Comment:
These images are foundational Swift images, and have nothing to do with Kitura by default, so we must communicate why this flow adds value to the user - it is not immediately clear why a user would need to generate a brand new project.

Suggestions:

+ In first line, talk about the easiest way to get started (with a new app, or an existing app that needs custom docker files) is Kitura Init, but reference the sections below for the users who just want to play around with our supported base images - this is an important use case, and they should not get wrapped up in this tool. 
+ It is not obvious why a Dockerfile-tools is necessary, or how it adds value to my project - let's move this section before the steps (or in the midst of them), that tells 'what' this does before the steps of 'how' to do it.
+fix `alsp`